MapleServer2/Packets/FieldObjectPacket.cs

```diff
@@ -21,19 +22,19 @@ public static Packet LoadPlayer(IFieldObject<Player> fieldPlayer)
             PacketWriter pWriter = PacketWriter.Of(SendOp.PROXY_GAME_OBJ);
             pWriter.WriteEnum(ProxyGameObjMode.LoadPlayer);
             pWriter.WriteInt(fieldPlayer.ObjectId);
-            pWriter.WriteLong(player.AccountId);
             pWriter.WriteLong(player.CharacterId);
+            pWriter.WriteLong(player.AccountId);
             pWriter.WriteUnicodeString(player.Name);
             pWriter.WriteUnicodeString(player.ProfileUrl);
             pWriter.WriteUnicodeString(player.Motto);
             pWriter.WriteByte();
             pWriter.Write(fieldPlayer.Coord);
             pWriter.WriteShort(player.Levels.Level);
-            pWriter.WriteEnum(player.Job); // short
-            pWriter.WriteEnum(player.JobCode);
-            pWriter.WriteInt();
-            pWriter.WriteInt();
-            pWriter.WriteInt();
+            pWriter.WriteShort((short) player.Job);
+            pWriter.WriteShort((short) player.JobCode);
```

Comment:
This should be using `WriteEnum`, not sure why changed.

Reply:
Because it's needed to write a short and most other places Job is used as int.